vimeo/vimeo-networking-java
# Support new API version 3.4.2

#### Issue
VA-3735

#### Summary
Bump API version from `3.4.1` -> `3.4.2`

Per API Changelog:

## 3.4.2 (2018-06-02)
* `account` in the `User` representation has been deprecated for `membership.type`.
* `badge` in the `User` representation has been deprecated for `membership.badge`.

#### Added

- The `User` representation has added the following fields:
    - `membership`
    - `membership.badge`
    - `membership.badge.alt_text`
    - `membership.badge.text`
    - `membership.badge.type`
    - `membership.badge.url`
    - `membership.display`
    - `membership.subscription`
    - `membership.subscription.renewal`
    - `membership.subscription.renewal.display_date`
    - `membership.subscription.trial`
    - `membership.subscription.trial.status`
    - `membership.type`


#### Removed

- The `User` representation has removed the following fields:
    - `account`
    - `badge`
    - `badge.alt_text`
    - `badge.text`
    - `badge.type`
    - `badge.url`
    

#### How to Test
Test apps should compile and run.
Vimeo App should correctly show badges and membership information after this change.
